examples/privileged-action-gate/README.md

```
@@ -0,0 +1,57 @@
+# privileged-action PR-gate (runnable, offline)
+
+An agent reaches a GitHub MCP server through the enforcing proxy and tries `github.add_deploy_key` on
+`acme/prod-app` — a privileged in-application write. The proxy decides **per call, before it
+forwards**, and writes a replayable `assay.enforcement_decision.v0` record. Everything here runs
+against a local mock: no real credentials, no real GitHub call.
+
+```bash
+./run.sh
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)